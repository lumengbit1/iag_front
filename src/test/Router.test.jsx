import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import thunk from 'redux-thunk';
import { createStore, applyMiddleware } from 'redux';
import { combineReducers } from 'redux-immutable';
import Router from '../router/routers';
import eventList from '../reducers/reducer';

const store = createStore(combineReducers({ value: eventList }), applyMiddleware(thunk));

describe('Route Test', () => {
  it('Route Test', () => {
    const component = renderer
      .create(
        <Provider store={store}>
          <MemoryRouter>
            <Router />
          </MemoryRouter>
        </Provider>,
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
