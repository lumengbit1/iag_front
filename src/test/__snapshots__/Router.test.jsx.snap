// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Route Test Route Test 1`] = `
<div
  className="sc-iCoGMd bKHGAZ"
>
  <div
    className="sc-pNWdM hlbGIy"
  >
    <div
      className="sc-fujyAs bAoGlv"
    >
      Results
    </div>
    <div
      className="sc-jrsJWt hFErOw"
    />
  </div>
  <div
    className="sc-pNWdM hlbGIy"
  >
    <div
      className="sc-fujyAs bAoGlv"
    >
      Saved Properties
    </div>
    <div
      className="sc-jrsJWt hFErOw"
    />
  </div>
</div>
`;
